app/views/admin/topical_events/show.html.erb

```diff
@@ -1,51 +1,53 @@
-<% page_title @topical_event.name %>
-
-<% content_for :current_tab do %>
-  <table class="<%= model_name %> table table-bordered add-top-margin">
-    <tr>
-      <th>Summary</th>
-      <td class="summary"><%= @topical_event.summary %></td>
-    </tr>
-
-    <tr>
-      <th>Description</th>
-      <td class="description"><%= @topical_event.description %></td>
-    </tr>
-
-    <tr>
-      <th>Details</th>
-      <td class="details"><%= topical_event_contents_breakdown(@topical_event) %></td>
-    </tr>
-
-    <% if @topical_event.is_a?(TopicalEvent) %>
-      <tr>
-        <th>Duration</th>
-        <td class="duration">
-          <% if @topical_event.start_date && @topical_event.end_date %>
-            <%= @topical_event.start_date %> to <%= @topical_event.end_date %>
-          <% end %>
-        </td>
-      </tr>
-    <% end %>
-
-    <% if @topical_event.respond_to?(:featured_links) %>
-      <tr>
-        <th>Featured links</th>
-        <td class="featured-links">
-          <% if @topical_event.featured_links.any? %>
-            <%= render 'shared/featured_links', links: @topical_event.featured_links %>
-          <% else %>
-            <em>None</em>
-          <% end %>
-        </td>
-      </tr>
-    <% end %>
-  </table>
-
-
-  <div class="form-actions">
-    <%= link_to "Edit", [:edit, :admin, @topical_event], class: "btn btn-lg btn-primary"%>
-  </div>
-<% end %>
-
-<%= render partial: 'tab_wrapper', locals: { model: @topical_event } %>
+<% content_for :page_title, @topical_event.name %>
+<% content_for :title, @topical_event.name %>
+<% content_for :title_margin_bottom, 4 %>
+
+<p class="govuk-body"><%= view_on_website_link_for @topical_event, class: "govuk-link" %></p>
```

Comment:
should this all be in 2/3 width?  probs one for Nik.

<img width="813" alt="image" src="https://github.com/alphagov/whitehall/assets/42515961/43cf1f39-0657-4cfc-87a1-c61fe56385f8">

Reply:
Have looked over the width issue with Nik and decided to stay with full width here. The narrower width starts to look a bit awkward at narrow, non-mobile, widths.